From: SeongJae Park <sj@kernel.org>
Subject: Docs/admin-guide/damon/usage: trivial fixups for DAMOS_MIGRATE_{HOT,COLD} documetnation
Date: Tue, 18 Jun 2024 14:36:29 -0700

Fixup below three trivial issues in DAMOS_MIGRATE_{HOT,COLD} usage
documentation[1,2].

1. Add target_nid on sysfs files hierarchy
2. Fix scheme directory section to mention target_nid file
3. Explain target_nid before apply_interval_us

Link: https://lkml.kernel.org/r/20240618213630.84846-2-sj@kernel.org
Fixes: c100fc71858a ("Docs/damon: document damos_migrate_{hot,cold}") # mm-unstable [1]
Link: https://lore.kernel.org/20240614030010.751-8-honggyu.kim@sk.com # [2]Signed-off-by: SeongJae Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/admin-guide/mm/damon/usage.rst |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

--- a/Documentation/admin-guide/mm/damon/usage.rst~docs-damon-document-damos_migrate_hotcold-fix
+++ a/Documentation/admin-guide/mm/damon/usage.rst
@@ -78,7 +78,7 @@ comma (",").
     │ │ │ │ │ │ │ │ ...
     │ │ │ │ │ │ ...
     │ │ │ │ │ :ref:`schemes <sysfs_schemes>`/nr_schemes
-    │ │ │ │ │ │ :ref:`0 <sysfs_scheme>`/action,apply_interval_us
+    │ │ │ │ │ │ :ref:`0 <sysfs_scheme>`/action,target_nid,apply_interval_us
     │ │ │ │ │ │ │ :ref:`access_pattern <sysfs_access_pattern>`/
     │ │ │ │ │ │ │ │ sz/min,max
     │ │ │ │ │ │ │ │ nr_accesses/min,max
@@ -289,21 +289,21 @@ schemes/<N>/
 ------------
 
 In each scheme directory, five directories (``access_pattern``, ``quotas``,
-``watermarks``, ``filters``, ``stats``, and ``tried_regions``) and two files
-(``action`` and ``apply_interval``) exist.
+``watermarks``, ``filters``, ``stats``, and ``tried_regions``) and three files
+(``action``, ``target_nid`` and ``apply_interval``) exist.
 
 The ``action`` file is for setting and getting the scheme's :ref:`action
 <damon_design_damos_action>`.  The keywords that can be written to and read
 from the file and their meaning are same to those of the list on
 :ref:`design doc <damon_design_damos_action>`.
 
-The ``apply_interval_us`` file is for setting and getting the scheme's
-:ref:`apply_interval <damon_design_damos>` in microseconds.
-
 The ``target_nid`` file is for setting the migration target node, which is
 only meaningful when the ``action`` is either ``migrate_hot`` or
 ``migrate_cold``.
 
+The ``apply_interval_us`` file is for setting and getting the scheme's
+:ref:`apply_interval <damon_design_damos>` in microseconds.
+
 .. _sysfs_access_pattern:
 
 schemes/<N>/access_pattern/
_
